index.html

```diff
@@ -5,8 +5,124 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="pege-header">
+      <nav class="navigation">
+          <img src="logo.svg" alt="Город Седона" width="40" height="30">
+        <ul class="navigation-list">
+           <li class="navigaton-item">
+             <a class="navigation-link" href="#">Главная</a>
+             </li>
```

Comment:
```suggestion
           </li>
```

ну в общем, всю эту кривоту нужно выровнять

Reply:
запустишь бьютифаер и всё ок будет